packages/strapi-plugin-content-manager/admin/src/containers/EditView/DeleteLink/utils/select.js

```diff
@@ -1,18 +1,37 @@
+import { isEmpty } from 'lodash';
 import useDataManager from '../../../../hooks/useDataManager';
 import useEditView from '../../../../hooks/useEditView';
 
 function useSelect() {
-  const { initialData, isCreatingEntry, isSingleType, slug, clearData } = useDataManager();
+  const {
+    initialData,
+    isCreatingEntry,
+    isSingleType,
+    slug,
+    clearData,
+    hasDraftAndPublish,
+    modifiedData,
+  } = useDataManager();
   const {
     allowedActions: { canDelete },
   } = useEditView();
 
+  let property = {};
+
+  if (hasDraftAndPublish) {
+    const isDraft = isEmpty(modifiedData.published_at);
+
+    property = isDraft ? { status: 'draft' } : { status: 'published' };
```

Comment:
Same here, `eventProperty` or `trackerEvent`. To make this variable more understandable.